frontend/src/features/multisig/multisigSlice.js

```diff
@@ -185,6 +215,43 @@ export const signTx = createAsyncThunk(
   }
 );
 
+export const verifyAccount = createAsyncThunk(
+  "multisig/verifyAccount",
+  async (data, { rejectWithValue, dispatch }) => {
+    try {
+      const token = await window.wallet.signArbitrary(
+        data.chainID,
+        data.address,
+        VERIFICATION_MESSAGE
+      );
+      try {
+        const response = await multisigService.verifyUser({
+          address: data.address,
+          signature: token.signature,
+          salt: 10,
+          pubKey: JSON.stringify(token.pub_key),
+        });
+        return {
+          response: response,
+          token: token,
+        };
+      } catch (error) {
+        return rejectWithValue(
+          error?.response?.data?.message || error?.message || SOMETHING_WRONG
```

Comment:
Instead of "something," you can define the error message as "error in verifying account" or "can't verify account."